src/components/ProductPage/ProductListing/ProductListing.jsx

```diff
@@ -0,0 +1,66 @@
+import React, {useState} from "react";
+import "./ProductListing.scss";
+import productData from "./Product.js";
+const ProductListing = () => {
+	console.log(productData);
+	const [statusCart, setstatusCart] = useState(false);
+	return (
+		<div className='productlisting-container'>
+			<div className='spacer-3rem'></div>
+			<div className='grid-3-columnlayout'>
+				{productData &&
+					productData.length &&
+					productData.map((product) => (
```

Comment:
Question: What does the productData.length helps in ?

Reply:
Actually i'm checking data array is empty or not empty. When every no data means length equal to zero. That time didn't go into inside map function. I'm adding Safe side considering edge case. :)